src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -0,0 +1,122 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import type * as OnyxTypes from '@src/types/onyx';
+import withPolicyAccessOrNotFound from './withPolicyAccessOrNotFound';
+import type {WithWorkspaceAccessProps} from './withPolicyAccessOrNotFound';
+
+type PolicyForList = {
+    value: string;
+    text: string;
+    keyForList: string;
+    isSelected: boolean;
+    rightElement: React.ReactNode;
+};
+
+type WorkspaceCategoriesOnyxProps = {
+    /** Collection of categories attached to a policy */
+    policyCategories: OnyxEntry<OnyxTypes.PolicyCategories>;
+};
+
+type WorkspaceCategoriesPageProps = WorkspaceCategoriesOnyxProps & WithWorkspaceAccessProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES>;
+
+function WorkspaceCategoriesPage({policyCategories}: WorkspaceCategoriesPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [selectedCategories, setSelectedCategories] = useState<string[]>([]);
+
+    const categoryList = useMemo<PolicyForList[]>(
+        () =>
+            Object.values(policyCategories ?? {}).map((value) => ({
+                value: value.name,
+                text: value.name,
+                keyForList: value.name,
+                isSelected: selectedCategories.includes(value.name),
+                rightElement: (
+                    <View style={styles.flexRow}>
+                        <Text style={[styles.disabledText, styles.alignSelfCenter]}>{value.enabled ? translate('workspace.common.enabled') : translate('workspace.common.disabled')}</Text>
+                        <View style={styles.p1}>
```

Comment:
Does the screenshot from [design doc](https://github.com/Expensify/App/pull/36324#issuecomment-1945851412) has more pading between the status and arrow right icon?

<img width="286" alt="image" src="https://github.com/Expensify/App/assets/29673073/d273ab05-f3fd-431a-bc8b-ed9e1842dd0c">

<img width="150" alt="image" src="https://github.com/Expensify/App/assets/29673073/c299ac74-5b00-4c76-9f75-bf0517dd4e06">

Reply:
fixed, thanks!